docs/background/plot_bgcube.py

```diff
@@ -0,0 +1,30 @@
+import matplotlib.pyplot as plt
+from astropy.units import Quantity
+from astropy.coordinates import Angle
+from gammapy.background import CubeBackgroundModel
+from gammapy import datasets
+
+filename = '../test_datasets/background/bg_cube_model_test.fits'
+filename = datasets.get_path(filename, location='remote')
+bg_model = CubeBackgroundModel.read(filename, format='table')
+
+fig, axes = plt.subplots(nrows=1, ncols=3)
+fig.set_size_inches(30., 8., forward=True)
+
+# plot images
+axes[0] = bg_model.plot_image(energy=Quantity(0.5, 'TeV'), ax=axes[0])
+axes[1] = bg_model.plot_image(energy=Quantity(50., 'TeV'), ax=axes[1])
+
+# plot spectra
```

Comment:
The example file still looks weird (background rate increasing with energy).
Can you change to the other one we talked about?